### ISSUE TITLE
Pagination - Use appropriate button or link element

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1261]
CENG_3333

Docs: https://react-magma.cengage.com/version/4.3.0/api/pagination/

Success Criteria: https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

The pagination component uses buttons within it, however there are many cases where these bring users to a new page. If that is the case, then these should be links vs buttons. It could be a good idea to ensure that when buttons like this are used they are to perform button actions, otherwise if these behave like links they should be updated to use link elements.

Orion - How do we support both scenarios in the same component?
